src/OpenTelemetry/Trace/Configuration/OpenTelemetrySdk.cs

```diff
@@ -0,0 +1,76 @@
+﻿// <copyright file="OpenTelemetrySdk1.cs" company="OpenTelemetry Authors">
```

Comment:
```suggestion
// <copyright file="OpenTelemetrySdk.cs" company="OpenTelemetry Authors">
```

Reply:
> This is going to be a real pain, many many people will forgot the null check

I don't think so. but anyway we'll watch the usage of the API. 

> Is there any pending work to support non-string values as activity tags?

No. Tags support strings only. OT support numbers/Bool which can still be used as strings.